Code/Tools/SceneAPI/SceneBuilder/SceneSystem.cpp

```diff
@@ -28,6 +29,13 @@ namespace AZ
 
         void SceneSystem::Set(const SDKScene::SceneWrapperBase* scene)
         {
+            static constexpr const char* s_UseOldAssimpBehaviorKey = "/O3DE/Preferences/SceneAPI/UseOldAssimpBehavior";
```

Comment:
IMHO, it makes more sense to enable an optional flag to default to false, not true. Can we change this to 'UseNewAssimpBehavior` and negate the conditions below? That also will remove the need to check the return type, since if the registry key is omitted, it wont set the referenced boolean value, and you default it to false.

Reply:
Applied, thanks